src/interface_py/h2o4gpu/util/gpu.py

```diff
@@ -149,6 +156,18 @@ def cudaresetdevice(gpu_id, n_gpus):
         from ctypes import c_int
         lib.cudaresetdevice(c_int(gpu_id), c_int(n_gpus))
 
+def cudaresetdevice_bare(n_gpus):
```

Comment:
Minor nitpick - could we rename it from `_bare` to something like `_current` (or something describing the fact it will reset the current process' device)?

Reply:
It's _bare() in sense that no checks or other cuda calls.  Needs to be bare bones.